src/xdocs/config_misc.xml

```diff
@@ -1541,6 +1541,85 @@ This is a sample text file. // ok, this file is not specified in the config.
       </subsection>
     </section>
 
+    <section name="NoCodeInFile">
+      <p>Since Checkstyle 8.33</p>
+
+      <subsection name="Description" id="NoCodeInFile_Description">
+        <p>
+          Checks whether file contains code.
```

Comment:
The description of the check is confusing. Will any file with a single/multi-line comment be considered empty? Or just a file with there is nothing except for some comments within?
Could you please make it more clear?

Reply:
I have specified the conditions in `Files which are considered to have no code:`.